openedx/core/djangoapps/user_api/accounts/tests/test_api.py

```diff
@@ -123,6 +123,30 @@ def test_update_error_validating(self):
                 {"profile_image": {"has_image": "not_allowed", "image_url": "not_allowed"}}
             )
 
+        # Check the various language_proficiencies validation failures.
```

Comment:
I see, though some of these validation errors bubble up from the `LanguageProficiencySerializer` (the middle two), and some come from the `AccountLegacyProfileSerializer`, so I'll keep one of each.